src/pages/homePage.ts

```diff
@@ -0,0 +1,13 @@
+import { Page } from "@playwright/test";
+export default class HomePage {
+
+constructor(public page: Page) {
+}
+   async clickOnSpecialHotMenu() {
+        await Promise.all([
+            this.page.waitForNavigation({ waitUntil: "networkidle" }),
+            
+            this.page.click("(//span[contains(text(),'Special')]/../..)[2]")

```

Comment:
locator should be in a separate variable

Reply:
made changes